recipe/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+export CFLAGS="-D__STDC_LIMIT_MACROS -D__STDC_CONSTANT_MACROS"
```

Comment:
Might be worth adding the SO answer link in a comment above in case anyone wonders why this is here. Included the short link below.

ref: https://stackoverflow.com/a/30472875